_targets.R

```diff
@@ -109,14 +109,14 @@ list(
     rbind(old_field_notes, mWater_field_notes)
```

Comment:
can we//do we have a dummy mWater_API.yml in the mWater_collate folder with the necessary formatting for it to work?

Reply:
Question: Do we want to just keep the true mWater yaml in the data folder or in the `mWaterCollate` folder? It has no real sensitive information (api creds, sensor data, etc) just field note data

Reply:
I think we should put it in the mWaterCollate folder so that we stay consistent with what we did for the api_pull. Sam, do you think that we will need to have a readme like what was done for the hydrovu api credentials?

Reply:
If we do, it can be very short essentially explaining what mWater is (link to it), how we use it to record our field notes and then to contact me if they have additional questions or would like to duplicate our field notes. 
The techs did create a table in the SOP with how to fill out each section if we want to include that